groups/sig-docs/groups.yaml

```
@@ -17,14 +17,13 @@ groups:
       WhoCanPostMessage: "ANYONE_CAN_POST"
       ReconcileMembers: "true"
     owners:
-      - jorgec@vmware.com
-      - kaitlynbarnard10@gmail.com
-      - paris.pittman@gmail.com
       - killen.bob@gmail.com
-      - zcorleissen@linuxfoundation.org
-    members:
-      - codyclark@google.com
-      - vonguard@gmail.com
+      - onlydole@gmail.com
+      - tim+cncf@scalefactory.com
```

Could I have:
```suggestion
      - tim+kubernetes@scalefactory.com
```

Also I'm not sure if Google groups supports `+` signs in email addresses - it might not.

---

I'd be somewhat surprised if it didn't. We could run a test?